### PR TITLE
docs(SDK-4092): Release cadence for 7.0.2

### PR DESCRIPTION
**Pull request by CTLalit**
https://wizrocket.atlassian.net/browse/SDK-4092

### CODE REVIEW

File: docs/CTCORECHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 7.0.2 (October 10, 2024)
+
+#### New Features
+* Adds support for custom handshake domain configuration in android manifest
```
Comment (Anush-Shand) [inline docs/CTCORECHANGELOG.md:6]: Should we also add changelog related to json definition of custom templates?

**Review by Anush-Shand — APPROVED** [review]

File: docs/CTCORECHANGELOG.md (hunk 1 above)
Comment (darshanclevertap) [reply docs/CTCORECHANGELOG.md:6]: I think we should

**Review by darshanclevertap — APPROVED** [review]

File: docs/CTCORECHANGELOG.md (hunk 1 above)
Comment (CTLalit) [reply docs/CTCORECHANGELOG.md:6]: According to @vasct and @nzagorchev it will not be used by native SDK.
Will confirm with them once again.

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Pull request merged**